packages/paste-core/layout/stack/__tests__/stack.test.tsx

```diff
@@ -120,18 +125,6 @@ describe('Stack Customization', () => {
     expect(renderedBox).toHaveAttribute('data-paste-element', 'STACK');
   });
 
-  it('should set the data-paste-attribute attribute for Stack Children to "STACK_CHILD"', (): void => {
-    const {container} = render(
-      <CustomizationProvider baseTheme="default">
-        <Stack orientation="vertical" spacing="space0">
-          <p>One</p>
-          <p>Two</p>
-        </Stack>
-      </CustomizationProvider>
-    );
-    expect(container.querySelector('[data-paste-element="STACK_CHILD"]')).toBeInTheDocument();
```

Comment:
issue: This is technically a public styling API. We can't really take it away anymore, that contract is set for customer apps and their custom styling.